### DISCUSSION TITLE
How to make Ampache 'read only'?

### DISCUSSION THREAD

**Discussion by Dadoosh** [ampache/ampache#3271] [Q&A] [answered]
Hello everybody,

I'm trying out Ampache and I would like it not to interfer with any folders and (audio) files inside its catalogue. As far as I have found out, playlists, user info etc. get written to the DB - perfect, just what I want! But is there a way that I can be absolutely certain that Ampache will not ever write to files (e.g. mp3 tags) or move them around? What settings do I need to adjust to achieve this?

Thanks in advance,
Dadoosh

-------------------------------------------
_Background info:
I curate my music library on my local file system and want to keep it that way. All changes get synced by Nextcloud to my online server which is where Ampache should live in the future. I want it to simply work with what it's being given and not mess up my NC sync. I will add a cronjob that makes Ampache update its catalogues once or twice a day. That should be enough for my purposes._

**Comment by kuzi-moto** [comment, marked as answer]
I'm pretty sure that by default Ampache is read-only. I think the three main config options found in `ampache.cfg.php` that will determine if files at modified are `write_id3` and `write_id3_art`, `write_tags`, all false by default.

**Reply by lachlan-00** [reply]
Ampache is primarily a file reader.
You can write tags/move/manipulate files but it's not automatic or required

**Reply by Dadoosh** [reply]
I had hoped it was as simple as that, but I wanted to be certain. Much appreciated, thanks!
